ConfigView.section.vpn_pp=Perfect Privacy için VPN Yardımcısı

check.port.every.mins=Bağlantı Noktası Yönlendirmeyi Denetleme ve Güncelleme Sıklığı (dakika olarak)
button.verify.addresses=Ortak adres(leri)i doğrula
section.portforwarding.title=Bağlantı Noktası Yönlendirme ve IP Bağlama
button.check.now=Şimdi Denetle

vpn.ip.regex=VPN IP Düzenli İfade Eşleşmesi
vpn.port.forwarding={base.product.name}''nin bağlantısı için bağlantı noktalarını, algılanan VPN''in Bağlantı Noktası Yönlendirmesine uyacak şekilde otomatik olarak güncelle

default.routing.not.vpn.network.splitting=Eğer Ağ Böleni'yseniz, bu kasıtlı olabilir.
default.routing.not.vpn.network.splitting.unbound=Eğer bağlantı noktalarınız yönlendirilirse (yönlendiricinizde ya UPnP ya da Bağlantı Noktası Yönlendirmesi aracılığıyla), {base.product.name} ana internet bağlantınızdan TCP/HTTP bağlantılarını hala alabilir.

pp.port.forwarding.get.failed=Bağlantı Noktası Yönlendirme alınamadı.

pp.port.in.registry=Kayıt defterinde depolanan bağlantı noktası %1
pp.port.calculated=%1 olarak hesaplanan Varsayılan Bağlantı Noktası Yönlendirmesi

pp.bound.good={base.product.name} UDP %1 adresine, diğer adıyla %2 arayüzüne bağlı
pp.bound.bad={base.product.name} UDP, %1 arayüzünün Perfect Privacy olmayan adresine bağlı
pp.nonvuze.probable.route={base.product.name} olmayan muhtemelen %1, diğer adıyla %2 üzerinden yönlendiriliyor.
pp.same.as.vuze={base.product.name} ile aynı.
pp.not.same=UYARI, {base.product.name} ile aynı değil\!
pp.vuze.unbound=Uyarı\: {base.product.name} UDP bağlı değil (Bunu düzeltmeye çalışacağız)
pp.vuze.loopback={base.product.name} UDP geri döngüde (Durdurma Anahtarı Devrede veya yanlış bağlama adresi/arayüzü)
pp.found.bindable.vpn=Büyük olasılıkla VPN'in yerel IP'si olan üzerinde %1 bağlanabilir adresi bulundu
pp.possible.vpn=%2 üzerinde %1 adresindeki muhtemelen Perfect Privacy
pp.assuming.vpn=VPN farz ediliyor. 
pp.same.address=Yukarıdaki adresle aynı \:)
pp.not.same.address=UYARI, yukarıdaki ile aynı değil\!
pp.not.same.future.address=UYARI, bağlamaya çalışacağımız ile aynı değil\!
pp.interface.not.found=Perfect Privacy'nin ağ arayüzü bulunamadı.  Belki VPN bağlantınız mı kesildi?
pp.nat.error=Nat Cihazları\: Alınamayan -> %1
pp.vpn.ip.detect.fail=VPN'in yerel IP'si belirlenemiyor
pp.already.bound.good=Mükemmel, {base.product.name} zaten %1 adresine bağlı
pp.bind.complete.triggered=Alınan bağlama uygulandı olayı
pp.change.binding={base.product.name}''nin %1 adresine, diğer adıyla %2 arayüzüne bağlanmasını değiştir
pp.changed.port=Çekirdek %1 bağlantı noktası %2 arayüzünden %3 arayüzüne değiştirildi

pp.indicator.warn=??
pp.indicator.ok=TAMAM
pp.indicator.bad=HATALI

pp.topline.warn=VPN UYARILARI VAR
pp.topline.ok=VPN TAMAM\!
pp.topline.bad=VPN DOĞRU OLARAK AYARLI DEĞİL
